src/stlink-lib/chipid.c

```diff
@@ -962,7 +961,7 @@ struct stlink_chipid_params *stlink_chipid_get_params(uint32_t chipid) {
   //  fprintf (stderr, "getparams: %x\n", chipid);
   for (params = devicelist ; params != NULL ; params = params -> next)
     if (params->chip_id == chipid) break;
-  
+
```

Comment:
```suggestion
  
```

Reply:
I'd prefer to avoid white space where not needed for improved readability.